definitions/contested/json/CaseEventToFields/CaseEventToFields.json

```diff
@@ -3887,7 +3887,9 @@
     "PageDisplayOrder": 3,
     "PageColumnNumber": 1,
     "FieldShowCondition": "applicantRepresented=\"Yes\"",
-    "ShowSummaryChangeOption": "Y"
+    "ShowSummaryChangeOption": "Y",
+    "CallBackURLMidEvent": "${CCD_DEF_COS_URL}/case-orchestration/amend-application-app-sol",
+    "RetriesTimeoutURLMidEvent": "3,4,5"
   },
```

Comment:
These names are fine.  But as jthmcts has already left a comment and for my information, is there anything stopping us using more explicit names in endpoints?  For instance, I would have probably gone with amend-application-applicant-solicitor.

Reply:
Nothing really stopping us to be honest... I suppose it would be better to make these endpoints clear for a dev. I'll take a look at this.